config/locales/forms.yml

```diff
@@ -582,6 +578,13 @@ en:
         actual_salary_html: Actual salary (optional<span class='govuk-visually-hidden'> field</span>)</span>
         benefits_html: Additional allowances (optional<span class='govuk-visually-hidden'> field</span>)</span>
         salary: Annual or full-time equivalent (FTE) salary
+      publishers_job_listing_working_patterns_form:
+        working_patterns_details_html: How many hours or days does the role cover? (optional<span class='govuk-visually-hidden'> field</span>)</span>
```

Comment:
It's simpler to use `, form_group: { classes: "optional-field" }`, which is a class we added so that we did not have to include html in the translations.

Reply:
I thought we moved away from that? 😕  Paging @alexbowen

Reply:
Moved away from which one? The approach I've gone with or David's? 😅